### PR TITLE
feat: support for waiting on stamp to be usable

### LINKED ISSUE #650: Support for waiting on postage stamp to be usable
It would be good to have a way for the user to easily wait until a new postage stamp is usable. I can see few ways:

 1. Add an option to `createPostageStamp()` method that would block the Promise until the stamp is usable and only then returning the stamp ID
 2. Utility function

### PR DESCRIPTION
**Pull request by AuHau**
Closes #650

### CODE REVIEW

File: src/bee-debug.ts [hunk 1]
```diff
@@ -511,15 +512,13 @@ export class BeeDebug {
       throw new BeeArgumentError(`Depth has to be at most ${STAMPS_DEPTH_MAX}`, depth)
     }
 
-    if (options?.gasPrice) {
-      assertNonNegativeInteger(options.gasPrice)
-    }
+    const stamp = await stamps.createPostageBatch(this.getKy(options), amount, depth, options)
 
-    if (options?.immutableFlag !== undefined) {
-      assertBoolean(options.immutableFlag)
```
Comment (vojtechsimetka) [inline src/bee-debug.ts:519]: Why did you remove the immutable option?

**Review by vojtechsimetka — APPROVED** [review]

File: src/bee-debug.ts (hunk 1 above)
Comment (AuHau) [reply src/bee-debug.ts:519]: Because I have used the `assertPostageBatchOptions` that was not used before, but was created for this check and have these checks in function.

**Comment by bee-worker** [comment]
Release [v4.1.0](https://github.com/ethersphere/bee-js/releases/tag/v4.1.0) addresses this.
